api/migrations/0001_initial.py

```diff
@@ -1,104 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
I ran into some issues with my database while changing the models and ended up deleting all of the migrations and starting over again. I'm actually not familiar with what the best practices are here for committing migrations. Is it fine that the migrations folder is empty?